nise/yaml_generators/oci/oci_yaml_constants.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCI Yaml File Constants"""
+from dataclasses import dataclass
+from dataclasses import field
+
+from faker import Faker
+
+
+fake = Faker()
+OCI_TAGS = {
+    "storage": ["tags/new-tags.tarnished-tags", "tags/orcl-cloud.free-tier-retained"],
+    "compute": ["tags/free-form-tag", "tags/orcl-cloud.free-tier-retained"],
+    "database": ["tags/free-form-tag"],
+    "network": ["tags/free-form-tag"],
+}
```

Comment:
It probably wouldn't hurt to make this a frozen dataclass just to prevent accidental modification even though it's not used directly by test code.
```suggestion
@dataclass(frozen=True)
class OCITags:
    storage: t.Tuple[str] = ("tags/new-tags.tarnished-tags", "tags/orcl-cloud.free-tier-retained")
    compute: t.Tuple[str] = ("tags/free-form-tag", "tags/orcl-cloud.free-tier-retained")
    database: t.Tuple[str] = ("tags/free-form-tag",)
    network: t.Tuple[str] = ("tags/free-form-tag",)
```